docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -148,7 +148,7 @@ the correct STS endpoint.
 ## Step 4/5. Give Teleport permissions to assume roles
 
 Next, attach the following policy to the IAM role or IAM user the Teleport
-Application Service instance is using, which allows the Application Service to
+Database Service instance is using, which allows the Database Service to
```

Comment:
I think we should revert this back to `Application Service` as the DynamoDB guide is using application access for the entire guide ATM.